src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -68,6 +68,7 @@ public function getConfigTreeBuilder()
                 ->booleanNode('enable_nelmio_api_doc')->defaultValue(false)->info('Enable the Nelmio Api doc integration.')->end()
                 ->booleanNode('enable_swagger')->defaultValue(true)->info('Enable the Swagger documentation and export.')->end()
                 ->booleanNode('enable_swagger_ui')->defaultValue(class_exists(TwigBundle::class))->info('Enable Swagger ui.')->end()
+                ->booleanNode('enable_graphql')->defaultValue(false)->info('Enable GraphQL.')->end()
```

Comment:
I suggest `defaultValue(class_exists(GraphQL::class))` (as for Swagger UI the line before).